src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="/"
+      method="get"
+      class="search-form form-search-big"
+      data-qa="big"
+    >
+      <label for="search-input">
+        <input
+        class="input input-search-big"
+        type="text"
+        name="search-big"
+        id="search-big"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        >
```

Comment:
Check 5-th point from [checkllist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md). Fix all cases
```suggestion
        <input
          class="input input-search-big"
          type="text"
          name="search-big"
          id="search-big"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
```

Reply:
changed